showcase/src/app/shared/table-of-contents/table-of-contents.component.scss

```diff
@@ -0,0 +1,58 @@
+:host {
```

Comment:
The Table of Contents is not rendered completely in tablet:
![table-of-contents-overflow](https://user-images.githubusercontent.com/5698318/48053553-46e6fb00-e1ab-11e8-82bb-06c7f8b3516d.PNG)


and in mobile phone I think it would nice if we keep the same blue border on the left of the table, what do you think?
![table-of-content-mobile](https://user-images.githubusercontent.com/5698318/48053559-4a7a8200-e1ab-11e8-8726-d49013f9865c.PNG)